sched/task/task_activate.c

```diff
@@ -94,4 +94,11 @@ void nxtask_activate(FAR struct tcb_s *tcb)
     }
 
   leave_critical_section(flags);
+
```

Comment:
The task may switched at line 93

Reply:
Yes it maybe switched out thus the log comes late. it is fine if meaning of this log is "task is already activated". It is at current location because logs are telling facts, not possiblities. Normally the task in question doesn't show up thus we rely on these facts to judge the system behavior.

doing it at line 86 may be fine but upon log time the task hasn't get chance to run yet.

Reply:
since some backend channels of syslog() is implemented in blocking model, print log may cause context switching here, so we need to be more cautious when adding logs in active flow.